po/ja.po

```diff
@@ -85,11 +85,11 @@ msgstr "システム"
 
 #: ../src/ibus-skk-preferences.ui.in.h:5
 msgid "User"
-msgstr "ユーザー"
+msgstr "ユーザ"
```

Comment:
Which guidelines are you referring to?  I heard that the recent JIS standard (Z8301) rather suggests to keep the macron at the end if the original English word ends with ```-er```, ```-or```, or ```-ar```.

Reply:
I fixed it just because that; in [72<sup>th</sup> line in `po/ja.po`](https://github.com/ueno/ibus-skk/blob/b6bda34d1079c7ef4fb1849c71ee68482f2f9537/po/ja.po#L72), it sees "ユーザ," but in [88<sup>th</sup> line in it](https://github.com/ueno/ibus-skk/blob/b6bda34d1079c7ef4fb1849c71ee68482f2f9537/po/ja.po#L88), it sees "ユーザー", not "ユーザ."

The two words are inconsistent. If you think "ユーザー" is right, please fix "ユーザ" in [72<sup>th</sup> line](https://github.com/ueno/ibus-skk/blob/b6bda34d1079c7ef4fb1849c71ee68482f2f9537/po/ja.po#L72).

Reply:
I see. Yes, I prefer the form with a macron. Would you mind updating the PR?

Reply:
Of course.
Now I [update](https://github.com/cmplstofB/ibus-skk/commit/b9acba05e12565b72789547cbdd3c3c45bbb7a12) `po/ja.po`.

---

I am grateful to you for your awesome program, IBus-SKK, thanks!